### DISCUSSION TITLE
examples of how to use it ?

### DISCUSSION THREAD

**Discussion by noor-tg** [umbrellio/laravel-ltree#78] [Q&A]
there is no documentation about how to use this package.

**Comment by noor-tg** [comment]
for example do I need only `path` column with type `ltree` in postgresql table ? or I need some other additional columns like `parent_id` .... ?

**Reply by pvsaintpe** [reply]
Generally speaking, it was planned that you would use only parent_id inside the project, and the formation of the path will be entrusted to our Services, which you can also use.

When a node is deleted, when a node is changed / moved, when the parent_id is changed, the path will be automatically updated.

You can use path to search and filter through SQL using predefined scopes or write your own queries.

If you have difficulties installing, configuring and using our extension, I am ready to provide you with demos, or examples of how we use it in our project.

Unfortunately, there are a lot of opportunities there and we can’t post any example in the documentation, because everything is individual, but alas, it is impossible to cover all the possibilities of ltree.

**Reply by noor-tg** [reply]
simple example to use tree hierarchical data (for example nested comments) and associated with blog post would be enough .

but thank you . I will try to use it . and maybe add pull request for example too.

**Reply by mandixx** [reply]
Is there any example that you guys can provide ? -

**Reply by noor-tg** [reply]
> Is there any example that you guys can provide ? -

I made my own solution. with postgresql & laravel eloquent. (it is not published)
